src/core/utils/composeKey.ts

```diff
@@ -1,5 +1,5 @@
 // TODO(DakEnviy): Do not use react-query in core
```

Comment:
Do you need todo?

Reply:
Yeah, just a reminder to find another function for hashing in the future